frontend/src/v5/ui/themes/theme.ts

```diff
@@ -181,6 +182,7 @@ export const theme = createTheme({
 			dark: COLOR.SECONDARY_DARK,
 			mid: COLOR.SECONDARY_MID,
 			light: COLOR.SECONDARY_LIGHT,
+			lighter: COLOR.SECONDARY_LIGHTER,
```

Comment:
Have you discussed this colour with George?

Reply:
I spoke to George, he said not to create a new colour in the palette for this, rather, to have it hardcoded in the icon itself